packages/pangea-node-sdk/src/utils/utils.ts

```diff
@@ -135,7 +138,7 @@ export function getCustomSchemaTestToken(environment: string) {
   return process.env[name] || "";
 }
 
-export function getFileUploadParams(file: string | Buffer): FileScan.ScanFileParams {
+export async function getFileUploadParams(file: string | Buffer): Promise<FileScan.ScanFileParams> {
```

Comment:
This is a breaking change to a public API so this will have to be a major release.